.github/workflows/codeql-analysis.yml

```diff
@@ -2,6 +2,7 @@ name: "Code scanning - action"
 
 on:
   push:
+  pull_request:
     paths-ignore:
```

Comment:
Should the filter still run on `push` or is it required in order to have the base commit to run without filtering?

Reply:
That's a good question. It is currently required to have the base commit run without filtering so that when you open a PR we definitely have results for the current `HEAD` of the base branch to compare against.

One thing you could do is limit both `push` and `pull_request` to the branches that you'll make PRs against (typically just `main`).